tasks/main.yml

```diff
@@ -48,3 +48,22 @@
     name: consul
     state: reloaded
   when: consul_service_definition.changed or consul_service_removal.changed
+
+- name: Upload a service to put consul under maintenance at shutdown
+  template:
+    src: consul-maint-before-shutdown.service.j2
+    dest: /lib/systemd/system/consul-maint-before-shutdown.service
+    owner: root
+    group: root
+    mode: "0644"
```

Comment:
```suggestion
    mode: "0660"
```

Reply:
Not sure about this change, if I create a systemd service I end up with `"0644"`

```
root@i-0c539d980708b2e2a:~# systemctl edit --force --full ahoguin.service
root@i-0c539d980708b2e2a:~# ls -la /etc/systemd/system/
...
-rw-r--r--  1 root root  194 Apr 10 15:58 ahoguin.service
```